sound/soc/soc-core.c

```diff
@@ -1656,9 +1657,14 @@ static void soc_check_tplg_fes(struct snd_soc_card *card)
 			dai_link->platforms->name = component->name;
 
 			/* convert non BE into BE */
-			dai_link->no_pcm = 1;
-			dai_link->dpcm_playback = 1;
-			dai_link->dpcm_capture = 1;
+				dai_link->no_pcm = 1;
```

Comment:
alignment

Reply:
Thanks @lyakh I must have some coffee before pushing this.